### ISSUE TITLE
Delete Icon

### ISSUE THREAD

**Issue by enguerran** [cozy/cozy-ui#341]
The dustbin icons are not the same, I told it on mattemost, I've been answered to open an issue:

cozy-drive/photos
![](https://i.imgur.com/YYxPLVB.png)
cozy-contacts
![](https://i.imgur.com/4oHjujB.png)

Code is:
```js
<Icon icon="delete" />
```

**Comment by GoOz** [comment]
Ô almighty @Claiw, as the only one truth holder, which one is the one?

**Comment by enguerran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [(2) ETU je peux supprimer un contact depuis sa fiche](https://trello.com/c/eD8KjGbM/724-2-etu-je-peux-supprimer-un-contact-depuis-sa-fiche)

**Comment by ptbrowne** [comment]
I hope it is the first :)

**Comment by Claiw** [comment]
Bonjour, Hello, Hola,

It is the first icon !

**Comment by GoOz** [comment]
After further investigations, both icons are actually the same. Same SVG code. 
But I noticed that even on https://cozy.github.io/cozy-ui/react/#icon where there is several instance of the same icon (like delete, destroy and trash) the rendering is different on Firefox but exactly the same on Chrome & Safari. I believe then that it's a Firefox bug (unless @enguerran you didn't take those screenshots with Firefox but I'm pretty sure you do). Still trying to figure out any solution but so far nothing…

**Comment by GoOz** [comment]
Found it! 🎉

**Comment by enguerran** [comment]
Both are taken with firefox
![](https://i.imgur.com/UMokPyW.png)

**Comment by enguerran** [comment]
On cozy-drive it is in-app code
On cozy-contacts it is cozy-ui code

**Comment by GoOz** [comment]
Yes, sorry I forgot to mention that the first in used in a background CSS and the second as inline SVG, that's why, they're different in the rendering but still the same SVG code behind. 
But the main issue is the bad rendering of inline SVG on Firefox, and I found a cure.

**Comment by enguerran** [comment]
I just did the chromium test: I confirm that the `cozy-ui` button with `<Icon icon="delete" />` is correct.
